### DISCUSSION TITLE
opensnitch-ui fails to launch on archlinux with gnome 44.1

### DISCUSSION THREAD

**Discussion by molitona** [evilsocket/opensnitch#935] [General]
upgrading to gnome 44.1 on arch with new version of python too (3.11.3) the opensnitch-ui fails to launch!

![Screenshot from 2023-05-11 00-37-44](https://github.com/evilsocket/opensnitch/assets/35532337/d9be5bc2-c269-4625-b8f2-434bb367c165)

**Comment by molitona** [comment]
─ opensnitchd -version                                                                                                            1.6.0-rc5

**Comment by gustavo-iniguez-goya** [comment]
Hi @molitona ,

See if this path exists: `/usr/lib/python3.11/*/opensnitch/` where * can be site-packages or dist-packages.
Did you upgrade to python 3.11 recently? Probably the installation is still at `/usr/lib/python3.10/site-packages/opensnitch/`

**Comment by molitona** [comment]
Hi @gustavo-iniguez-goya,

The dependencies are still under /usr/lib/python3.10 
and when upgraded arch their was new version of python which is 3.11
upgraded python (3.10.10-1 -> 3.11.3-1)

there is opensnitch/ and opensnitch_ui-1.6.0rc5.dist-info/ under /usr/lib/python3.10/site-packages. should i move them to /usr/lib/python3.11/ ?

**Reply by gustavo-iniguez-goya** [reply]
Actually, I'm not sure. I think that if you reinstall the GUI, it should be installed for your "python3" version, which in this case it should be python3.11.

**Reply by molitona** [reply]
the copy worked fine for me. thank you for help!

**Reply by kwkvyuha** [reply]
red hat 10.1
无法 depsolve 事务； 发现 1 问题：
 问题: 冲突的请求
  - nothing provides python3-qt5 needed by opensnitch-ui-1.7.2-1.noarch from @commandline
  -怎么办？

**Reply by gustavo-iniguez-goya** [reply]
@kwkvyuha upgrade to the v1.8.0 version. Many distros have started to remove PyQt5 from their repos.
